keyvalue/server/pom.xml

```diff
@@ -46,7 +46,12 @@
     </dependency>
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-server</artifactId>
+      <artifactId>alluxio-core-server-master</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-server-worker</artifactId>
```

Comment:
BTW, the convention is already violated in this pom as `alluxio-core-server` depends on `alluxio-core-common` and `alluxio-core-client` and neither is included in the list of dependencies